lang/sbcl-bootstrap/Portfile

```diff
@@ -75,6 +75,9 @@ if {${build_arch} eq "ppc"} {
 
 master_sites        sourceforge:project/sbcl/sbcl/${version}
 
+# https://sourceforge.net/p/sbcl/mailman/message/58774549/
+master_sites-append https://macos-powerpc.org/compilers/sbcl/boot/:macppc
```

Comment:
Whose site is this? It's not the SBCL project.

Reply:
I control this domain and server for it, and Sergey fills it with data.

Reply:
That makes it difficult to trace the provenance of such bootstrap binaries etc.

Reply:
Do you have better idea? SBCL can't be bootstrap by so old SBCL anymore.

Reply:
Can it be bootstrapped from a Common Lisp interpreter in C? CLISP say?

Reply:
The last time I tried it can't. Thus, I not sure that modern enough Clisp or CCL is supported by PowerPC as well.

Reply:
Is the problem that you can't compile clisp for powerpc? It's pure C.

Reply:
The problem that SBCL can't be bootstraped on macOS by CLisp. I've tried it a year or two ago last time and it had filed. I had opened a few bugs in upstream but nobody reacted.

Reply:
I think I could light a fire under upstream if necessary.

Reply:
(BTW, what specifically fails?)

Reply:
@pmetzger Perry, this is a temporary measure, which was suggested by SBCL upstream. They do not have powerpc hardware, and have to rely on us.

There is no requirement for this to be a binary provided by myself and Kirill though. If @kencu can help to compile it on his powerpc machine and upload it somewhere to MacPorts hosting, that would be even better. We need a standard build here, which upstream can reasonably trust.

Reply:
I think it's better to bootstrap it off of CLISP. In theory they're both ANSI Common Lisp and it should be possible. What's needed is detail on how and why this is failing.

Reply:
@ryandesign Ryan, any chance of having a limited (controlled by you) buildbot for powerpc set up? That would have solved the problem, though we would need a two-stage bootstrap for powerpc for a while.

Reply:
I really still think it's better to boostrap off CLISP. There are probably bugs right now and upstream is almost certainly interested in fixing them. I know enough lispheads that I could make the right contacts.

Reply:
CLISP is portable and written in C. It does not include a compiler, it's a pure interpreter, and upstream is responsive.

Reply:
If I understood Kirill correctly, the problem is not to build Clisp, but to use it to build SBCL.

@catap Could you refer us to the issues with Clisp on macOS? Even regardless of this ticket, it will be nice to have those fixed.

Reply:
clisp should be able to build SBCL, but the maintainer doesn't really have a modern CI for all this stuff and is probably not regularly testing it. But he'll probably take patches.

Reply:
After much time having passed, and having noted that SBCL maintainers have made a conditional offer to host the PPC binaries in the futures once MacPorts takes a step towards earning trust <https://sourceforge.net/p/sbcl/mailman/message/58774549/>, I am for merging this PR as-is, over @pmetzger's preference for a clisp bootstrap mechanism.  Were someone to produce patches to use CLISP (or ABCL) to bootstrap, I would sponsor those patches.